### ISSUE TITLE
Add export/import Word and PDF in online file editor

### ISSUE THREAD

**Issue by regislutter** [zetura/syncr#1]
Add export/import of Word or PDF in the online file editor:
- Word library: [phpDocx](http://www.phpdocx.com/)
- PDF library: [FPDF](http://www.fpdf.org/?lang=en)

**Comment by YannickGagnon** [comment]
May I suggest this library for managing Word Documents

One benefit is that it's FREE :+1: 

https://github.com/PHPOffice/PHPWord

**Comment by regislutter** [comment]
Yes, I've seen it, but I made the choice of phpDocx, because it have "Document merging" functionality. Which would be really great to compare copydeck versions ;)
